src/components/cartPage/cartInfo/cartInfo.tsx

```diff
@@ -0,0 +1,109 @@
+import InputText from "@/elements/inputText/inputText";
+import { useState } from "react";
+import styles from "./cartInfo.module.scss";
+
+interface Element {
+  id: number;
+  title: string;
+  platform: string;
+  amount: number;
+  price: string;
+}
+
+type SelectCards = string[];
+
+interface CartArray {
+  cards: Element[];
+  deleteItem: (value: string[]) => void;
+  updateCards: (value: Element[]) => void;
+}
+
+const CartInfo = ({ cards, deleteItem, updateCards }: CartArray): JSX.Element => {
+  const [selectValues, setSelectValues] = useState<SelectCards>([]);
+
+  const selectCards = (checkboxes) => {
+    const checkBoxesValue: string[] = [];
+    for (let index = 0; index < checkboxes.length; index++) {
+      if (checkboxes[index].checked) {
+        checkBoxesValue.push(checkboxes[index].value);
+      }
+    }
+    setSelectValues(checkBoxesValue);
+  };
+
+  const changeAmount = (newAmount: string, idElement: number) => {
```

Comment:
You can use one method for updating the platform and amount. Just provide the name as an argument and use it.